### ISSUE TITLE
[Accessibility][Container][Sample Content] Insufficient color contrast on Background Image and Nested container examples

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1730]
**Steps to Reproduce**
1. In Google Chrome, view Background Image and Nested container examples.
2. Test color contrast of text over a dark section of the background image using a color contrast tool.

**Current Behavior**
Text on the background images has various levels of color contrast ratios given the nature of the changing background colors, but many areas fail to meet the 4.5:1 contrast ratio required of 18 point (24 CSS pixels or less) text.

**Expected behavior/code**
Text of 24 CSS pixels or less meets a 4.5:1 contrast ratio with the background; text larger than that meets a 3:1 contrast ratio. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision and/or limited perception of color.

**WCAG SC**
1.4.3 Contrast

**Screenshot**
![The Background Image example, with arrows pointing to dark text with very low contrast over dark parts of the background image.](https://user-images.githubusercontent.com/86849351/128246204-38c867b4-11f5-4267-9515-1a8a3108989b.png)

**Possible Solution**
Use background colors with strong contrast to the text color and/or use high contrast text shadows. Prompt authors to use images that will provide adequate contrast.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
